src/server/routes.ts

```diff
@@ -1,12 +1,86 @@
-import express from "express";
+import express, { json } from "express";
 import cors from "cors";
+import Controller from "./controller/index";
+import { Trade } from "./types/index";
+
+const environment: string | undefined = process.env.PORT;
+const controller = new Controller();
 
 const app = express();
 
+app.use(express.urlencoded());
+app.use(json());
```

Comment:
**Comment**: If we are going to say `express.urlencoded()`, we should be consistent in our naming conventions.

Reply:
Unsure of what you mean. Could you clarify? I could bind express.urlencoded() to a variable if that is what you are referring to.

Reply:
`urlencoded` belongs to class `Express`. Either use `import { json, urlencoded }` followed by `app.use(urlencoded())` and `app.use(json())` or a flat Express import, and use `app.use(express.urlencoded())` and `app.use(express.json())`.